clients/filesystem-hadoop3/src/main/java/com/datastrato/gravitino/filesystem/hadoop/GravitinoVirtualFileSystem.java

```diff
@@ -285,27 +285,22 @@ private FileStatus convertFileStatusPathPrefix(
     return fileStatus;
   }
 
-  private NameIdentifier extractIdentifier(URI virtualUri) {
-    Preconditions.checkArgument(
-        virtualUri
-            .toString()
-            .startsWith(GravitinoVirtualFileSystemConfiguration.GVFS_FILESET_PREFIX),
-        "Path %s doesn't start with scheme prefix \"%s\".",
-        virtualUri,
-        GravitinoVirtualFileSystemConfiguration.GVFS_FILESET_PREFIX);
-
-    if (StringUtils.isBlank(virtualUri.toString())) {
-      throw new InvalidPathException("Uri which need be extracted cannot be null or empty.");
+  @VisibleForTesting
+  NameIdentifier extractIdentifier(URI virtualUri) {
+    String virtualPath = virtualUri.toString();
+    if (StringUtils.isBlank(virtualPath)) {
+      throw new InvalidPathException(
+          virtualPath, "Uri which need be extracted cannot be null or empty.");
```

Comment:
Can we use `Preconditions` for simplicity?

Reply:
done